sentinelhub/geometry.py

```diff
@@ -531,105 +531,3 @@ def _parse_geometry(geometry: Union[Polygon, MultiPolygon, dict, str]) -> Union[
             raise ValueError(f"Supported geometry types are polygon and multipolygon, got {type(geometry)}")
 
         return geometry
-
-
-class BBoxCollection(_BaseGeometry):
```

Comment:
only deprecate for now. If we decide to do aggressive removal in the next version, we'll throw it out then. Until then I want the option to do a minor release without breaking code.

Reply:
Done.